.NET/Microsoft.Recognizers.Text.DateTime/Constants.cs

```diff
@@ -59,7 +59,7 @@ public static class Constants
         public const string Comment_WeekOf = "WeekOf";
         public const string Comment_MonthOf = "MonthOf";
 
-        public const string Comment_ContainFeb29th = "ContainFeb29th";
+        public const string Comment_DoubleTimex = "DoubleTimex";

```

Comment:
Please add a code comment here, like:
// Tag to mark cases where the specifc resolution timex depends on future or past values.